terraform/dev-desktops/_terraform.tf

```diff
@@ -45,3 +45,8 @@ provider "aws" {
   region  = "us-east-1"
 }
 
+provider "azurerm" {
+  features {}
```

Comment:
What does auth for this look like locally? I sort of think maybe it makes sense to split into separate directories - it can be annoying to need credentials for something you're not changing.

I wonder if we could pivot off of AWS credentials to fetch temporary azure credentials automatically.

Reply:
That's a good point. Happy to split the module into `dev-desktops-aws` and `dev-desktops-azure`.

We haven't created a `team-members-azure` module yet to manage user accounts on Azure. With the limited knowledge that I have about Azure, it seems that we'd need to set up Azure AD and provision accounts there. That seemed a bit much just for spawning two virtual machines, so I've used the root account and the Azure CLI to authenticate.